### DISCUSSION TITLE
How to have localized file names

### DISCUSSION THREAD

**Discussion by Mia-Ameisbichler** [facebook/docusaurus#10346] [Q&A]
Is there any way to have localized file names in the respective folders? Like docs/hello.md and i18n/fr/docusaurus-plugin-content-docs/bonjour.md and then link them together. 
I expermiented with ids and slugs, setting both of them to the english variant, but it seems that when a localized file with the name of the original doesn't exist the original content will be displayed instead.

**Comment by slorber** [comment]
We don't support that

We might support later `docs/hello.fr.md`, but it more complex than I thought (PR: https://github.com/facebook/docusaurus/pull/9700)
